src/lotus/message/mod.rs

```diff
@@ -0,0 +1,68 @@
+//! This module contains the various response types to be used byt the lotus api.
+
+use std::str::FromStr;
+
+use anyhow::anyhow;
+use cid::Cid;
+use serde::{Deserialize, Serialize};
+
+pub(crate) mod chain;
+pub(crate) mod mpool;
+pub(crate) mod state;
+pub(crate) mod wallet;
+
+/// Exec actor parameters
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct ExecParams {
+    pub code_cid: Cid,
+    pub constructor_params: Vec<u8>,
+}
```

Comment:
@cryptoAtwill I think this are now part of the fvm_utils and can be removed from the agent, right? It may have been left behind from your PR.

Reply:
I'll remove this

Reply:
thx thx!